acceptance/tests/options/config_file/cache_migrate_from_facter_3.rb

```diff
@@ -0,0 +1,79 @@
+test_name 'ttls configured custom facts files creates cache file and reads cache file' do
```

Comment:
The third case `run Facter 4 with no config file` is not covered.

Reply:
Updated